cloudslang-compiler/src/test/resources/task-args-one-liner/flow_expression_contains_reserved_characters.yaml

```diff
@@ -18,7 +18,7 @@ flow:
   workflow:
     - CheckWeather:
         do:
-          ops.check_Weather: city = city_name if city_name != 'abc' else 'efg', country = str('Wonderland')
+          ops.check_Weather: city = ${ city_name if city_name != 'abc' else 'efg' }, country = ${ 'Wonderland' }
```

Comment:
I think that `"Wonderland"` should remain a string to keep the original functionality